### ISSUE TITLE
localhost points to 0.0.0.0

### ISSUE THREAD

**Issue by MustafaCan** [StevenBlack/hosts#5]
It should point to 127.0.0.1 or not included. (most systems handle this automatically)

**Comment by StevenBlack** [comment]
Closing.  We're now pointing to 0.0.0.0 always.

**Comment by rcyeh** [comment]
No no no. As described in issue #5, issue #15, and issue #21, by providing `0` (an invalid address) as the mapping for `localhost`, you are changing a standard assumption for this special name, and some software may not handle that correctly. See https://en.wikipedia.org/wiki/Localhost . Every one of your upstream sources with `localhost` maps it to the standard `127.0.0.1` loopback address and you should too, unless you have a good reason not to do that --- and that should be made explicit in your comments.

In general, web advertisements will *not* try to obtain their data from `localhost`, because most host computers are not serving advertisements or malware; so mapping `localhost` to the standard `127.0.0.1` will normally not increase the advertisements or malware experienced. This assumption may be violated if you already have malware installed on your computer.

In short, that one line
```
0 localhost
```
doesn't offer any real protection and tends to break things.
